django/test/client.py

```diff
@@ -518,7 +538,10 @@ def get(self, path, data=None, follow=False, secure=False, **extra):
     def post(self, path, data=None, content_type=MULTIPART_CONTENT,
              follow=False, secure=False, **extra):
         """Request a response from the server using POST."""
-        response = super().post(path, data=data, content_type=content_type, secure=secure, **extra)
+        response = super().post(
+            path, data=data, content_type=content_type,
+            secure=secure, **extra
+        )
```

Comment:
I think you should revert this unrelated change, and also similar changes in `put()`, `patch()`, and `delete()`.

Reply:
This is an artifact from an earlier draft.

I am happy to revert it if desired. @carltongibson any opinion?

Reply:
Yes. Good spot @felixxm. 

I think we try and keep the change footprints as small as possible. It's unrelated so best to leave it out. 

(We could do it in a separate change, but the original is not so long as to offend — to my eye at least...)

Reply:
I wouldn't say it warrants a change. It stays under 120 chars so in my mind is fine.